### ISSUE TITLE
Feature Request: Smart Search (finder) plugin

### ISSUE THREAD

**Issue by jojo12** [slabbi/uddeIM#83]
With Joomla!4+ ended com_search. If you update from Joomla!3 you still have search plugins and they work. 
If you install Joomla 4+ new, then you don't get com_search!
Because Joomla says, that finder plugins are smarter and more intelligent than search plugins. So the question is: Is it possible to create a uddeIM smart search plugin? Make this sense? And how to do this?

**Comment by joomod** [comment]
?? hmm,
I don't know how the uddeim_searchbot.php exactly works.
hope slabbi can give an answer

**Comment by slabbi** [comment]
Correct. Please don't ask me about the Joomla finder. My last coding for Joomla was for J4 beta. I am using Wordpress for about eight years now.

**Comment by joomod** [comment]
I'm trying to find out for finder
